gazebo_ros2_control/src/gazebo_ros2_control_plugin.cpp

```diff
@@ -244,49 +247,62 @@ void GazeboRosControlPlugin::Load(gazebo::physics::ModelPtr parent, sdf::Element
     return;
   }
 
-  // Load the RobotHWSim abstraction to interface the controllers with the gazebo model
+  std::unique_ptr<hardware_interface::ResourceManager> resource_manager_ =
+    std::make_unique<hardware_interface::ResourceManager>();
+
   try {
+    // We cannot use (for now) the <ros2_control> tag in the URDF, we need to call it manually.
```

Comment:
IMO, you do have access to it via the component parser: https://github.com/ros-controls/ros2_control/blob/master/hardware_interface/include/hardware_interface/component_parser.hpp